### PR TITLE
fix: Allow for configurable dist location

### LINKED ISSUE #213: Allow for configurable dist location

### PR DESCRIPTION
**Pull request by grshane**

### CODE REVIEW

File: outline.config.js [hunk 1]
```diff
@@ -1,18 +1,23 @@
+/* Set a different path for compiled assets
+ * if you change this value you must also change
+ * "outDir" in your tsconfig.json
+ */
+const destBasePath = 'dist';
```
Comment (mabry1985) [inline outline.config.js:5]: Do you not need to export this?

Comment (grshane) [reply outline.config.js:5]: No, because I am only using it locally in that file

Comment (mabry1985) [reply outline.config.js:5]: I saw it used below with outline.destBasePath, so I was thinking it had to be exported up here. Carry on.

**Review by mabry1985 — APPROVED** [review]
Straight forward change.

**Review by himerus — CHANGES_REQUESTED** [review]
Currently getting duplicate directories after clearing out my local and running both `yarn start` and `yarn build` locally. 

Screenshot of Code (2-1-22, 10-31-43 AM)

**Review by himerus — APPROVED** [review]

**Pull request merged**